test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenTransactSpecs.java

```diff
@@ -432,6 +439,46 @@ public HapiApiSpec balancesChangeOnTokenTransfer() {
 				);
 	}
 
+	public HapiApiSpec uniqueTokenTransfers() {
+		return defaultHapiSpec("UniqueTokenTransfers")
+				.given(
+						cryptoCreate(FIRST_USER).balance(0L),
+						cryptoCreate(SECOND_USER).balance(0L),
+						cryptoCreate(TOKEN_TREASURY).balance(0L),
+						newKeyNamed("supplyKey"),
+						tokenCreate(A_TOKEN)
+								.tokenType(TokenType.NON_FUNGIBLE_UNIQUE)
+								.supplyType(TokenSupplyType.INFINITE)
+								.initialSupply(0)
+								.supplyKey("supplyKey")
+								.treasury(TOKEN_TREASURY),
+						mintToken(A_TOKEN, List.of(ByteString.copyFromUtf8("memo"))),
+						tokenAssociate(FIRST_USER, A_TOKEN),
+						tokenAssociate(SECOND_USER, A_TOKEN)
+				).when(
+						cryptoTransfer(
+								movingUnique(1, A_TOKEN).between(TOKEN_TREASURY, FIRST_USER)
+						).via("cryptoTransferTxn"),
+						cryptoTransfer(
+								movingUnique(1, A_TOKEN).between(FIRST_USER, SECOND_USER)
+						)
+				).then(
+						getAccountBalance(TOKEN_TREASURY)
+								.hasTokenBalance(A_TOKEN, 0),
+						getAccountBalance(FIRST_USER)
+								.hasTokenBalance(A_TOKEN, 0),
+						getAccountBalance(SECOND_USER)
+								.hasTokenBalance(A_TOKEN, 1),
+
+						getTokenNftInfo(A_TOKEN, 1)
+								.hasSerialNum(1)
+								.hasMetadata(ByteString.copyFromUtf8("memo"))
+								.hasTokenID(A_TOKEN)
+								.hasAccountID(SECOND_USER),
+						getReceipt("cryptoTransferTxn").logged()
```

Comment:
We can get the `transaction record` and assert that there is a `nft` transfer in the `tokenTransferList`